packages/volto/src/components/theme/Search/Search.jsx

```diff
@@ -109,7 +109,11 @@ class Search extends Component {
     const options = qs.parse(this.props.history.location.search);
     this.setState({ currentPage: 1 });
     options['use_site_search_settings'] = 1;
-    this.props.searchContent('', options);
+    const { settings } = config;
+    this.props.searchContent('', {
+      b_size: settings.defaultPageSize,
```

Comment:
@davisagli I put a `console.log` in the constructor and it printed the `defaultPageSize`. I think it's better this way, because you'll have the code in one place.

Reply:
Should I refactor the code?
If yes, please elaborate on what you meant by `class defaultPageSize`? Isn't it a variable?

Reply:
@EshaanAgg I think @wesleybl wants it as a prop, or to extract it as a static field of the Search class component. Having it as a prop would provide some value. I don't see the other option, "extracting it as a field" as providing a lot of value except by making it more readable? I don't know. It would add a level of indirection.

Reply:
Got it!